src/rollback_to_stable/rts_btree.c

```diff
@@ -64,6 +64,11 @@ __rts_btree_abort_update(WT_SESSION_IMPL *session, WT_ITEM *key, WT_UPDATE *firs
         } else {
             /* Valid update is found. */
             stable_upd = upd;
+            __wt_verbose_level_multi(session, WT_VERB_RECOVERY_RTS(session), WT_VERBOSE_DEBUG_4,
+              WT_RTS_VERB_TAG_STABLE_UPDATE_FOUND "stable update found with txnid=%" PRIu64
+                                                  ", stable_timestamp=%s,  durable_timestamp=%s",
+              upd->txnid, __wt_timestamp_to_string(rollback_timestamp, ts_string[1]),
```

Comment:
Can you also print the `upd->flags` in a hex value as that provides the flag value and it can be helpful in deciding the following code flow?

Reply:
Done! [2ff1584](https://github.com/wiredtiger/wiredtiger/pull/8960/commits/2ff15849792bdb0a1de4f07f3f8cb187db8e72a2)